german/the-black-hack.md

```diff
@@ -316,20 +317,20 @@ Beim Würfeln, um festzustellen ob sich Eigenschaften erhöhen, würfeln Klerike
 
 ### Göttliche Anrufungen
 
-Ab der zweiten Stufe können Kleriker täglich eine gewisse Anzahl von Göttlichen Anrufungen durchführen, siehe den Abschnitt zu Zaubersprüchen.
+Ab der zweiten Stufe können Kleriker täglich eine gewisse Anzahl von göttlichen Anrufungen durchführen, siehe den Abschnitt zu Zaubersprüchen.
 
 ### Gebetbuch
 
-Kleriker beginnen das Spiel mit einem großen Gebetbuch, das zu Beginn 1W4 Zaubersprüche der Stufe 1 und 2 von der Liste Göttlicher Anrufungen enthält.
+Kleriker beginnen das Spiel mit einem großen Gebetbuch, das zu Beginn 1W4 Zaubersprüche der Stufe 1 und 2 von der Liste göttlicher Anrufungen enthält.
 
 ----
 
 ## Zauberer
 
 **Start-TP :** W4 + 4
 **TP pro Stufe/Erholung :** 1W4
-**Waffen & Rüstung :** Einhändiges Schwert, Stab
-**Angriffsschaden :** 1W4 / 1 Unbewaffnet oder Improvisiert
+**Waffen & Rüstung :** einhändiges Schwert, Stab
```

Comment:
Das würde aber nur in geschriebener Form funktionieren. Am Tisch wüsste ich nicht wie ich zwischen "Großen Schilden" und "großen Schilden" unterscheiden soll.